deepspeed/runtime/utils.py

```diff
@@ -856,3 +856,21 @@ def call_to_str(base, *args, **kwargs):
         name += ', '.join(f'{key}={repr(arg)}' for key, arg in kwargs.items())
     name += ')'
     return name
+
+
+def print_backward_tensors(tensor):
```

Comment:
IMHO, such util best belongs to: https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/utils/debug.py

And as you can see there are already all kinds of shortcuts there.

Reply:
@stas00, good point. 

@eltonzheng, can you please move to the debug file? Thanks!

Reply:
sure, thanks @stas00, will move it to the debug file.